.github/workflows/docker-publish.yaml

```diff
@@ -0,0 +1,56 @@
+name: Build Docker Image
+
+on: [workflow_dispatch, push]
```

Comment:
Before merging we should remove the `push` trigger. When would we want this job trigger? I would go for manual trigger only

Reply:
manual dispatch sounds ok!

Btw, are you still working on the caching (your comment above) or not really?

Reply:
I guess we won't run this job that often, so I would leave the caching for now

Reply:
sounds good.. I will merge now! Looks good to me